corehq/motech/repeaters/models.py

```diff
@@ -146,24 +146,15 @@
     UserPayloadGenerator,
 )
 
-# Retry responses with these status codes. All other 4XX status codes
-# are treated as InvalidPayload errors.
-HTTP_STATUS_4XX_RETRY = (
-    HTTPStatus.BAD_REQUEST,
-    # Traefik proxy returns (client error) 404 instead of (server error)
-    # 503 when it has not been configured with a catch-all router and is
-    # unable to match a request to a router.
-    # https://doc.traefik.io/traefik/getting-started/faq/#404-not-found
-    HTTPStatus.NOT_FOUND,
+# Back off, and retry responses with these status codes. All other
+# status codes are treated as InvalidPayload errors.
+HTTP_STATUS_BACK_OFF = (
     HTTPStatus.REQUEST_TIMEOUT,
-    HTTPStatus.CONFLICT,
```

Comment:
Just saying this out loud, but the dhis2entity repeater appeared to have a lot of conflict responses, which I imagine could result in a lot of invalid payloads that need to be manually retried.

Reply:
For what it's worth, it's the removal of 400 "Bad Request" that will impact DHIS2. The function that sends cases to DHIS2 as tracked entity instances goes through a workflow of requests, and it's those that are shown in the Remote API Logs. When there are conflicts that it can't resolve, then it will [treat the payload as a 400 response here](https://github.com/dimagi/commcare-hq/blob/aec092ac237b71d9f932d0c5d67a56f914b86dfa/corehq/motech/dhis2/entities_helpers.py#L77).